### PR TITLE
Update the documentation to the new structure

### PR DESCRIPTION
**Pull request by sachinira**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,42 +12,40 @@ management, role based access control etc to first and third-party Software as a
 Azure AD Client Connector allows you to perform operations on users and groups of an Azure AD. The operations are
```
Comment (shaniR) [inline README.md:12]: Shall we rephrase the above paragraph as: "Azure Active Directory(AD) is Microsoft’s multi-tenant, cloud-based directory and identity management service. It provides capabilities to enable Single Sign-On (SSO), multi-factor authentication, self-service password reset, privileged identity management, role-based access control, etc., to first and third-party Software as a Service (SaaS) applications.	"

Comment (shaniR) [reply README.md:12]: What is Azure AD Client Connector? Is it a term Azure uses?

Comment (shaniR) [reply README.md:12]: Let's rephrase this as: "Azure AD Client Connector allows you to perform operations on users and groups in an Azure AD using the v1.0 version of Microsoft Graph API. The client supports OAuth 2.0 authentication."

File: README.md [hunk 2]
```diff
@@ -12,42 +12,40 @@ management, role based access control etc to first and third-party Software as a
...
-  * [Oracle](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
- 
-  * [OpenJDK](https://adoptopenjdk.net/)
+For more information, go to the module(s).
+- [azure.ad](aad/Module.md)
+
+## Building from the source
+### Setting up the prerequisites
+1.  Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
+   > **Note:** Set the JAVA_HOME environment variable to the path name of the directory into which you installed
```
Comment (shaniR) [inline README.md:21]: Use markdown syntax for the note, if this cannot be added into a step.

File: aad/Module.md [hunk 3]
```diff
@@ -1,91 +1,33 @@
 ## Overview
-Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language 
-easily. It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
+Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language.
```
Comment (shaniR) [inline aad/Module.md:2]: Let;s use Active Directory (AD) in place of the first AD.

File: aad/Module.md [hunk 4]
```diff
@@ -1,91 +1,33 @@
 ## Overview
-Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language 
-easily. It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
+Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language. 
+It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent
```
Comment (shaniR) [inline aad/Module.md:3]: Do we need to use `` for the resource types. And I don't think we need to capitalize them as well.

File: aad/Module.md [hunk 5]
```diff
@@ -1,91 +1,33 @@
 ## Overview
-Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language 
-easily. It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
+Ballerina connector for Azure AD is connecting the Azure AD REST API in Microsoft Graph v1.0 via Ballerina language. 
+It provides capability to perform management operations on `User` and `Group` resources in an Azure AD tenent 
 (Organization).
 
 The connector is developed on top of Microsoft Graph is a REST web API that empowers you to access Microsoft Cloud
```
Comment (shaniR) [inline aad/Module.md:6]: Let's rephrase this as: " The connector uses the Microsoft Graph REST web API that provides the capability to access Microsoft Cloud service resources. This version of the connector only supports the operations on users and groups in an Azure AD"

File: aad/Module.md [hunk 6]
```diff
@@ -1,91 +1,33 @@
...
+    | User.ReadWrite.All            | Delegated | Create channels                           |
+    | Group.ReadWrite.All           | Delegated | Read and write all users' full profiles   |
 
 ## Quickstart
-## Create Azure AD User
-### Step 1: Import Azure AD Package
-First, import the ballerinax/msgraph_onedrive module into the Ballerina project.
+To use the Azure AD connector in your Ballerina application, update the .bal file as follows:
+### Step 1 - Import connector
+First, import the ballerinax/aad module into the Ballerina project.
```
Comment (shaniR) [inline aad/Module.md:26]: Let's remove the word First,

File: aad/Package.md [hunk 7]
```diff
@@ -1,19 +1,18 @@
 Connects to Azure AD from Ballerina
 
-### Package Overview
+## Package overview
 The `ballerinax/azure.ad` is a [Ballerina](https://ballerina.io/) connector for Azure AD.
```
Comment (shaniR) [inline aad/Package.md:4]: Shall we use the connector name here?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: aad/Package.md (hunk 7 above)
Comment (sachinira) [reply aad/Package.md]: Used the connector name

File: aad/Module.md (hunk 6 above)
Comment (sachinira) [reply aad/Module.md]: Removed

File: aad/Module.md (hunk 5 above)
Comment (sachinira) [reply aad/Module.md]: Fixed

File: aad/Module.md (hunk 4 above)
Comment (sachinira) [reply aad/Module.md]: Fixed

File: aad/Module.md (hunk 3 above)
Comment (sachinira) [reply aad/Module.md]: Added

File: README.md (hunk 1 above)
Comment (sachinira) [reply README.md]: The Client part is unnecessary. Added the suggestion

File: README.md (hunk 2 above)
Comment (sachinira) [reply README.md:18]: This is added as per the [common format agreed by the team](https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit#heading=h.96g5nsmszl71). The changes which will alter that format will be added in next release cycles

**Review by abeykoon — APPROVED** [review]

File: aad/client.bal [hunk 8]
```diff
@@ -16,19 +16,22 @@
 
 import ballerina/http;
 
-# Azure AD Client Object for executing drive operations.
+# This is connecting to the Microsoft Graph RESTful web API that enables you to access Microsoft Cloud service resources.
```
Comment (shaniR) [inline aad/client.bal:19]: The Azure Active Directory connector allows you to connect to the Microsoft Graph RESTful web API. It enables you to access Microsoft Cloud Services resources.

File: aad/client.bal [hunk 9]
```diff
@@ -16,19 +16,22 @@
...
-public client class Client {
-    http:Client httpClient;
-    Configuration config;
-
-    # Initializes the Azure AD connector client endpoint.
-    #
+public isolated client class Client {
+    final http:Client httpClient;
+    final readonly & Configuration config;
+    # Gets invoked to initialize the `connector`.
```
Comment (shaniR) [inline aad/client.bal:29]: Initializes the connector.

File: aad/client.bal [hunk 10]
```diff
@@ -242,17 +235,15 @@ public client class Client {
         return finalStream; 
     }
 
-    # Get groups that the group is a member of. This operation is transitive and will also include all groups that this 
-    # groups is a nested member of.
+    # Gets groups that the group is a member of. <br/> This operation is transitive and will also include all groups
```
Comment (shaniR) [inline aad/client.bal:238]: Is this the description provided by the AAD API?
